base/ca/src/main/java/com/netscape/cms/profile/common/EnrollProfile.java

```diff
@@ -373,8 +373,9 @@ public void setDefaultCertInfo(Request request) throws EProfileException {
             info.set(X509CertInfo.KEY,
                     new CertificateX509Key(X509Key.parse(new DerValue(dummykey))));
 
+            String dummySubjectName = this.getConfigStore().getBoolean("emptyDefaultSubjectName", true) ? "" : "CN=Dummy Subject Name";
```

Comment:
I think we should be able to access `mConfig` directly instead of calling `this.getConfigStore()`.

Reply:
Oh right, sorry, I missed that suggestion completely! I will revise in the morning.